File: day48_part_3/a1.py
def hash_map_example():

    user_settings = {
        "theme":"dark",
        "notifications":"enabled"
    }

    print(f"Theme: {user_settings.get("theme")}")


    if "theme" in user_settings:
        print("Theme is present")

    
    for key,value in user_settings.items():
        print(f"{key}: {value}")
        
    user_settings.pop("theme",None)
    print(f"After removing theme: {user_settings}")

if __name__ == "__main__":
    hash_map_example()

File: day48_part_3/a1.js
const userSettings ={
    theme:"dark",
    notifications:"enabled"
};


console.log("Theme:",userSettings.theme);

if("theme" in userSettings){
    console.log("Theme is set");
}

for(const [key,value] of Object.entries(userSettings)){
    console.log(`${key}: ${value}`);
    
}

delete userSettings.theme;
console.log("After removing theme:",userSettings);